tests/helix/send-to-helix-templatestests.targets

```diff
@@ -53,8 +53,9 @@
         <PreCommands>%(PreCommands) $(_ShellCommandSeparator) $(_EnvVarSetKeyword) &quot;CODE_COV_FILE_SUFFIX=-%(TestNameSuffix)&quot;</PreCommands>
         <PreCommands>%(PreCommands) $(_ShellCommandSeparator) $(_EnvVarSetKeyword) &quot;TEST_NAME_SUFFIX=%(TestNameSuffix)&quot;</PreCommands>
 
-        <PreCommands Condition="'$(OS)' == 'Windows_NT'">%(PreCommands) $(_ShellCommandSeparator) set &quot;TEST_ARGS=--filter category^^!=failing^&amp;FullyQualifiedName~%(Identity)&quot;</PreCommands>
-        <PreCommands Condition="'$(OS)' != 'Windows_NT'">%(PreCommands) $(_ShellCommandSeparator) export &quot;TEST_ARGS=--filter category!=failing&amp;FullyQualifiedName~%(Identity)&quot;</PreCommands>
+        <!-- append '.' to the name so only the test class with exactly that name is run-->
```

Comment:
What was happening here?

Reply:
Using `FullyQualifiedName~Aspire.Templates.Tests.StarterTemplateRunTests` matched `Aspire.Templates.Tests.StarterTemplateRunTests_PreviousTFM` too. I'll add this to the comment.